Controller/ConnectController.php

```diff
@@ -296,7 +297,8 @@ protected function getErrorForRequest(Request $request)
      */
     protected function getResourceOwnerByName($name)
     {
-        $ownerMap = $this->container->get('hwi_oauth.resource_ownermap.'.$this->container->getParameter('hwi_oauth.firewall_name'));
+        // TODO: fix firewall_names[0]
+        $ownerMap = $this->container->get('hwi_oauth.resource_ownermap.'.$this->container->getParameter('hwi_oauth.firewall_names')[0]);
```

Comment:
@stloyd Thanks a lot! I guess it will work same most of the time, and I guess there will be reasonable number of firewalls most of the time.